suites/pacific/rbd/tier-2_rbd_trash_purge.yaml

```diff
@@ -0,0 +1,97 @@
+# Tier2: Extended RBD trash related testing
+#
+# This test suite runs addition test scripts to evaluate the existing functionality of
```

Comment:
Can you be more specific on the tests being done in this test suite?

Reply:
Done.